cli/attestation.go

```diff
@@ -70,32 +120,261 @@ func (cli *CLI) NewGetAttestationCmd() *cobra.Command {
 }
 
 func (cli *CLI) NewValidateAttestationValidationCmd() *cobra.Command {
```

Comment:
The following fields should have default values:
- `host_data` - default value should be `0` (the field should be filled with zeros).
- `family_id` - default value should be `0` (the field should be filled with zeros).
- `image_id` - default value should be `0` (the field should be filled with zeros).
- `report_id` - default value should be `0` (the field should be filled with zeros).
- `report_id_ma` - default value should be `0` (the field should be filled with zeros).